persist-cli/src/main/java/io/ballerina/persist/cmd/Push.java

```diff
@@ -195,6 +209,17 @@ private void setupJdbcDriver() throws BalException {
             throw new BalException("Error in jdbc driver path : " + e.getMessage());
         }
     }
+    private void populatePlaceHolders(HashMap<String, String> persistConfigurations, String name,
+                                                         String projectName)
+            throws BalException {
+        configurations = SyntaxTreeGenerator.readToml(Paths.get(this.sourcePath, this.configPath), name);
+        for (String configKey : persistConfigurations.keySet()) {
+            if (persistConfigurations.get(configKey).replaceAll("\"", "").equals(
+                    String.format(PLACE_HOLDER_TEMPLATE, projectName, configKey))) {
+                persistConfigurations.put(configKey, configurations.get(configKey));
+            }
+        }
```

Comment:
What happens when user updates and messes up the template and the place holders are not matched from the Config toml?

Reply:
We must stop executing if the placeholder value can't resolve from the config.toml. And give proper error messages, including the placeholder value, which couldn't resolve.

Reply:
In this case we return `Persist client related config doesn't exist in Config.toml.You should add database configurations under <pkg name>.clients`